src/osdep/amiberry.cpp

```diff
@@ -2646,6 +2650,24 @@ void save_amiberry_settings(void)
 	snprintf(buffer, MAX_DPATH, "rom_path=%s\n", rom_path);
 	fputs(buffer, f);
 
+	snprintf(buffer, MAX_DPATH, "rp9_path=%s\n", rp9_path);
+	fputs(buffer, f);
+
+	snprintf(buffer, MAX_DPATH, "floppy_sounds_dir=%s\n", floppy_sounds_dir);
+	fputs(buffer, f);
+
+	snprintf(buffer, MAX_DPATH, "data_dir=%s\n", rom_path);
+	fputs(buffer, f);
+
+	snprintf(buffer, MAX_DPATH, "saveimage_dir=%s\n", rom_path);
+	fputs(buffer, f);
+
+	snprintf(buffer, MAX_DPATH, "savestate_dir=%s\n", rom_path);
+	fputs(buffer, f);
+
+	snprintf(buffer, MAX_DPATH, "screenshot_dir=%s\n", rom_path);
+	fputs(buffer, f);
```

Comment:
Note to self: fix these as midwan pointed out (see conversation)

Also testing if I can see my own comments.

Reply:
Resolved with https://github.com/midwan/amiberry/pull/864/commits/db9e0426329eafce12d2044232f1573aa86d02e2